### ISSUE TITLE
Editor component preview breaks with less-than "<" character

### ISSUE THREAD

**Issue by chrisgeary92** [netlify/netlify-cms#496]
**- Do you want to request a *feature* or report a *bug*?**

BUG

**- What is the current behavior?**

When using the WYSIWYG editor (non-markdown editor), adding less than characters `<` are converted to `&lt;` which breaks editor previews.

Example: `{{< youtube nthOpo5tQDk >}}` becomes `{{&lt; youtube nthOpo5tQDk >}}`

**- If the current behavior is a bug, please provide the steps to reproduce.**

Add the following editor component (this is the example YouTube component on the Netlify docs), and attempt to add the following shortcode in the WYSIWYG editor:

Shortcode: `{{< youtube nthOpo5tQDk >}}`

```
CMS.registerEditorComponent({
  id: "youtube",
  label: "Youtube",
  fields: [{name: 'id', label: 'Youtube Video ID'}],
  pattern: /^{{<\s?youtube (\S+)\s?>}}/,
  fromBlock: function(match) {
    return {
      id: match[1]
    };
  },
  toBlock: function(obj) {
    return '{{< youtube ' + obj.id + ' >}}';
  },
  toPreview: function(obj) {
    return (
      '<img src="http://img.youtube.com/vi/' + obj.id + '/maxresdefault.jpg" alt="Youtube Video"/>'
    );
  }
});
```

Alternatively, clone [chrisgeary92/netlify-cms-editor-issue-2](https://github.com/chrisgeary92/netlify-cms-editor-issue-2), update the repo in config.yml and try it out.

**- What is the expected behavior?**

Expecting `{{<` to not convert to `{{&lt;`

**Comment by erquhart** [comment]
Known issue, will be addressed by #254.

**Comment by chrisgeary92** [comment]
Sorry, didn't see this, thanks :)
